controllers/autoneg.go

```diff
@@ -377,8 +398,16 @@ func validateOldConfig(cfg OldAutonegConfig) error {
 	return nil
 }
 
-func validateNewConfig(cfg AutonegConfig) error {
-	// do additional validation
+func validateNewConfig(config AutonegConfig) error {
+	for _, cfgs := range config.BackendServices {
+		for _, cfg := range cfgs {
+			if cfg.InitialCapacity != nil {
+				if *cfg.InitialCapacity < 0 || *cfg.InitialCapacity > 100 {
```

Comment:
I figured that it's safer to throw a validation error in case an operator passes a capacity outside of the acceptable range, but we could also apply a forced default of either 0% (safer behavior) or 100% (current behavior) to avoid the error case